src/tools.ts

```diff
@@ -1,4 +1,5 @@
 import { TFile, renderMath, finishRenderMath, parseLinktext, resolveSubpath, getLinkpath } from "obsidian";
+import MathLinks from "./main";
```

Comment:
If I remove this import line, `npm run build` gives this error:
```
src/api.ts:14:32 - error TS2304: Cannot find name 'MathLinks'.

14     constructor(public plugin: MathLinks, public manifest: PluginManifest, public blockPrefix: string) {
                                  ~~~~~~~~~
```

So I think we should keep this. 

P.S. Unlike `build`, `npm run dev` yields no error without the line. 
Moreover, the current code has a lot of errors that are invisible with `dev` but visible with `build`.
It would be nice if they could be fixed in the near future.

```
Found 72 errors in 6 files.

Errors  Files
    15  src/links.ts:6
     3  src/main.ts:17
     6  src/preview.ts:5
    37  src/settings.ts:58
     6  src/supercharged.ts:3
     5  src/utils.ts:5
```

(Perhaps you are making releases manually with `npm run dev`?)

If you are interested, the following is the officially recommended way of making releases:

https://docs.obsidian.md/Plugins/Releasing/Release+your+plugin+with+GitHub+Actions

Reply:
Ohh, I see. Thanks for letting me know! I have a lot to learn...